### DISCUSSION TITLE
Continued updates?

### DISCUSSION THREAD

**Discussion by netsendjoe** [RadzPrower/Super-Metroid-Launcher#4] [Ideas]
It would be great to see an update to this launcher that brings over updates made to the Zelda 3 Launcher. Also tailoring it more toward Super Metroid since the initial release of this launcher seems more like a copy and paste. Specifically, I feel like Mode 7 and MSU-1 would be beneficial as well as any other development towards enhancing the gameplay.

**Comment by RadzPrower** [comment]
As far as I am aware, those updates have not been made to the Super Metroid project by snesrev.

The launcher is indeed a copy and paste of Zelda 3 Launcher because all it does is build the game and allow for editing some settings. There's very little tailoring to be done since snesrev's implementations are largely the same for both games.

**Comment by netsendjoe** [comment]
Oh. That makes sense, so it's more about updating the Super Metroid source
code to be able to have some quality of life modifications.
